clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverExecutor.scala

```diff
@@ -51,19 +50,14 @@ abstract class DeliverExecutor[CI <: DeliverableIndex](
   }
 
   override protected[dispatch] def buildMove(
-    metadata: moveCommand.index.MetadataType,
-    ioUtil: IoUtil
+    metadata: moveCommand.index.MetadataType
   ): Source[(moveCommand.index.MetadataType, immutable.Seq[IoOp]), NotUsed] = {
 
-    val baseStream = super
-      .buildMove(metadata, ioUtil)
-      .orElse(Source.single(metadata -> immutable.Seq.empty))
-
+    val baseStream = Source.single(metadata -> immutable.Seq.empty)
     baseStream.flatMapConcat {
```

Comment:
Shouldn't need the `baseStream` / `flatMapConcat` wrapper any more; could just call `buildDelivery(metadata.withWorkspaceName(...))`

Reply:
Thanks.